SPECS/cryptsetup/cryptsetup.spec

```diff
@@ -1,135 +1,146 @@
- 
+Summary:        Headers and libraries for using encrypted file systems
+Group:          Development/Libraries
+Requires:       %{name} = %{version}-%{release}
+Requires:       pkgconfig
+Provides:       cryptsetup-luks-devel = %{version}-%{release}
+
 %description devel
 The cryptsetup-devel package contains libraries and header files
 used for writing code that makes use of disk encryption.
- 
+
 %package libs
-Group: System Environment/Libraries
-Requires: json-c
-Summary: Cryptsetup shared library
-Provides: cryptsetup-luks-libs = %{version}-%{release}
- 
+Summary:        Cryptsetup shared library
+Group:          System Environment/Libraries
+Requires:       json-c
+Provides:       cryptsetup-luks-libs = %{version}-%{release}
+
 %description libs
 This package contains the cryptsetup shared library, libcryptsetup.
- 
+
 %package -n veritysetup
-Group: Applications/System
-Summary: A utility for setting up dm-verity volumes
-Requires: cryptsetup-libs = %{version}-%{release}
- 
+Summary:        A utility for setting up dm-verity volumes
+Group:          Applications/System
+Requires:       cryptsetup-libs = %{version}-%{release}
+
 %description -n veritysetup
 The veritysetup package contains a utility for setting up
 disk verification using dm-verity kernel module.
- 
+
 %package -n integritysetup
-Group: Applications/System
-Summary: A utility for setting up dm-integrity volumes
-Requires: cryptsetup-libs = %{version}-%{release}
- 
+Summary:        A utility for setting up dm-integrity volumes
+Group:          Applications/System
+Requires:       cryptsetup-libs = %{version}-%{release}
+
 %description -n integritysetup
 The integritysetup package contains a utility for setting up
 disk integrity protection using dm-integrity kernel module.
- 
+
 %package reencrypt
-Group: Applications/System
-Summary: A utility for offline reencryption of LUKS encrypted disks.
-Requires: cryptsetup-libs = %{version}-%{release}
- 
+Summary:        A utility for offline reencryption of LUKS encrypted disks.
+Group:          Applications/System
+Requires:       cryptsetup-libs = %{version}-%{release}
+
 %description reencrypt
 This package contains cryptsetup-reencrypt utility which
 can be used for offline reencryption of disk in situ.
- 
+
 %prep
 %setup -q -n cryptsetup-%{version}
 %patch0 -p1
 chmod -x misc/dracut_90reencrypt/*
- 
+
 %build
-%configure --enable-fips --enable-pwquality --enable-internal-sse-argon2 --with-crypto_backend=openssl --with-default-luks-format=LUKS2
+./autogen.sh
+%configure --enable-fips --enable-pwquality --enable-internal-sse-argon2 --with-default-luks-format=LUKS2 --disable-ssh-token
 make %{?_smp_mflags}
- 
+
 %install
 make install DESTDIR=%{buildroot}
-rm -rf %{buildroot}/%{_libdir}/*.la
- 
+find %{buildroot} -type f -name "*.la" -delete -print
+
 %find_lang cryptsetup
- 
+
 %post -n cryptsetup-libs -p /sbin/ldconfig
- 
 %postun -n cryptsetup-libs -p /sbin/ldconfig
- 
+
 %files
 %{!?_licensedir:%global license %%doc}
```

Comment:
Nit: macro definition will never be needed, since we set the license directory in all versions of Mariner.

Reply:
Thanks, I'll clean that up.